pool/pool.gno

```diff
@@ -26,11 +26,11 @@ func Mint(
 	tickUpper int32,
 	_liquidityAmount string, // uint128
 	positionCaller std.Address,
-) (string, string) { // uint256 x2
+) (string, string) { // uint256 x2== "0"
```

Comment:
comments can be removed, it is not that meaningful

Reply:
https://github.com/gnoswap-labs/gnoswap/pull/403/commits/92d7e60ed3167ede13cf8539d6d23631bb03dc5c